source/_docs/organization-dashboard.md

```diff
@@ -41,18 +41,36 @@ Removing site team members follows the same process.
 
 ## Filter Sites
 
-At the Sites tab, the left panel contains groups of filters for limiting the sites list. Filters will appear in each group as sites are added that match the filters. You can add custom tags by selecting the checkbox next to the site, and clicking **Tags** and **Add Tag(s)**, then entering the tag.
+At the Sites tab, the left panel contains groups of filters for limiting the sites list. Filters will appear in each group as sites are added that match the filters.
+
+### Service Level
+This lets you filter sites by their [site plan](/docs/site-plan/).
+
+### Tags
+You can add custom tags by selecting the checkbox next to the site, and clicking **Tags** and **Add Tag(s)**, then entering the tag.
 
 <div class="alert alert-info" role="alert">
 <h4 class="info">Note</h4>
 <p>Tags are case-sensitve.
 </p>
 </div>
+
 To remove tags, select the site(s) you want to remove and follow the procedure above, this time selecting **Remove Tag**, or by hovering over the tag and clicking the **x** that appears.
 
+### Upstream
+Use this filter to sort sites by their upstream. This includes both [Pantheon upstreams](/docs/start-state/#pantheon-upstreams) and [custom upstreams](/docs/custom-upstream/).
+
 ### Code Status
 This filter shows which sites have core updates available, which ones are up to date, and which ones are unknown (e.g., managed by Composer). At this time, Organizations with over 800 sites do not have access to this filter, as they will time out loading.
 
+### Status
+
+This filter shows any sites that are [frozen](/docs/platform-considerations/#inactive-site-freezing), or awaiting upgrade to the [Global CDN](/docs/global-cdn/).
+
+### User in Charge
+Filters sites by the [user in charge](/docs/change-management/#roles-and-permissions).
```

Comment:
Suggest relating user in charge role to site creation

Reply:
@rachelwhitton This is covered in the linked doc:
> This helps distinguish who created a site for Enterprise and EDU Organizations where members are allowed to spin up new Sandbox sites at will. 

Because this information isn't related to site filters directly, I would elect not to include that information here, to reduce duplicate information for future updates.

Reply:
Seems valuable to know what the role is in context of the filter - your call though, defer to you.

Reply:
This might need to be presented as `### User in Charge / Owner` - depending on org type you'd have one or the other